pkgs/native_toolchain_c/lib/src/cbuilder/cbuilder.dart

```diff
@@ -47,7 +47,7 @@ class CBuilder implements Builder {
   /// The filename will be decided by [BuildConfig.targetOS] and
   /// [OS.libraryFileName] or [OS.executableFileName].
   ///
-  /// File will be placed in [BuildConfig.outputDirectory].
+  /// File will be placed in [BuildConfig.outDirectory].
```

Comment:
Why the rename? We just renamed it from `outDir` to `outputDirectory`? why `outDirectory`?